cg/apps/crunchy/crunchy.py

```diff
@@ -327,12 +335,20 @@ def get_sbatch_path(log_dir: Path, compression: str, run_name: str = None) -> Pa
         # Only other option is "SPRING"
         return log_dir / "_".join([run_name, "decompress_spring.sh"])
 
-    def _submit_sbatch(self, sbatch_content: str, sbatch_path: Path):
+    def _submit_sbatch(self, sbatch_content: str, sbatch_path: Path, pending_path: Path):
         """Submit SLURM job"""
         LOG.info("Submit sbatch")
         if self.dry_run:
             LOG.info("Would submit sbatch %s to slurm", sbatch_path)
             return
+        LOG.info("Creating pending flag")
```

Comment:
do we need all of the logging?

Reply:
I love it!

Reply:
I find it pretty helpful when parsing the logs, there has been so many obstacles with this so I prefer to log a lot